lib/api/src/modules/stories.ts

```diff
@@ -439,12 +446,25 @@ export const init: ModuleFn = ({
       const { sourceType } = getEventMetadata(this, fullAPI);
 
       if (sourceType === 'local') {
+        const { storyId, storiesHash } = store.getState();
         const options = fullAPI.getCurrentParameter('options');
 
         if (options) {
           checkDeprecatedOptionParameters(options);
           fullAPI.setOptions(options);
         }
+
+        // create a list of related stories to be preloaded
+        const toBePreloaded = Array.from(
+          new Set([
+            api.findSiblingStoryId(storyId, storiesHash, 1, false),
+            api.findSiblingStoryId(storyId, storiesHash, -1, false),
```

Comment:
There's no need to preload the sibling stories of the same component, as they all come from the same CSF file, and thus will already be loaded. I guess there's no harm in doing so? Not sure.

Reply:
fair, I will change this.